zeroconf.py

```diff
@@ -1080,6 +1080,7 @@ def __init__(self, type, name, address=None, port=None, weight=0,
             self.server = server
         else:
             self.server = name
+        self._properties = {}
```

Comment:
Race condition in which field is used before it is declared.  Latent bug exposed bug new test case.